cmd/root.go

```diff
@@ -21,6 +21,7 @@ const (
 	cmdFlagKeyStack        = "stack"
 	cmdFlagKeyAddWebhook   = "add-webhook"
 	cmdFlagKeyAutoCodesign = "auto-codesign"
+	cmdFlagKeyAPIToken = "api-token"
```

Comment:
Did you forget to turn on go vet? This constant doesn't seem to be aligned properly.